docs/adding-new-dapps.md

```diff
@@ -1,3 +1,9 @@
+<style>
+td, th {
+   border: none!important;
+}
+</style>
```

Comment:
Let's remove these styles they don't appear to applied [on GitHub](https://github.com/valora-inc/dapp-list/blob/3ab8592b5a10a4de8f2161b28980d096a8a06d21/docs/adding-new-dapps.md).